openapi3/example_validation.go

```diff
@@ -0,0 +1,19 @@
+package openapi3
+
+import (
+	"context"
+)
+
+func ValidateExampleValue(ctx context.Context, input interface{}, schema *Schema) error {
+	opts := make([]SchemaValidationOption, 0, 3) // 3 potential opts here
+	opts = append(opts, VisitAsResponse())
+	opts = append(opts, VisitAsRequest())
```

Comment:
What are you trying to do with these options here and how do you think this gets tested?